postman/README.md

```diff
@@ -0,0 +1,38 @@
+# Postman code examples
+
+This directory contains examples for use with Postman
+
+| Script | Link |
+|--------|------|
+| Create and Schedule Broadcast|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/8c408d4c102f2152f5c5)|
+| Find Subscribers Across Lists|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/bffa24a4a56d16e06c6e)|
+| Get Broadcast Details|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/cdb69f4c7f3dafefd0d9)|
+| Get Broadcast Open Clicks|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/ba9d9c5722eaf0a8388e)|
+| Get Broadcast Stats|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/50e72a9d31ec072180d5)|
+| Get Clicks|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2ca986836d3bc2244e65)|
+| Get Integrations|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/0566aa365b24d75822e9)|
+| Get Tags|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/223a97960ea532ae59cd)|
+| Get Webforms Detail|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/a88306f31ebe6c8e1949)|
+| Manage Custom Fields|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/966f607e6a5b3b78751c)|
+| Manage Subscriber|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/23943d73ae00a6371eb5)|
+| Move Subscriber|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2ced13b02af75197126a)|
+
+Please follow this tutorial to import this [Postman Collection](https://www.getpostman.com/docs/collections)
+
+## Environment variables that need to be created in Postman:
+
+### For all requests
+
+- consumer_key
+- consumer_secret
+- access_token
+- token_secret
+
+### To search for a subscriber by email across all lists
```

Comment:
This is also used to add a new subscriber in Manage Subscriber

Reply:
Added